tests/codebase/test_code_quality.py

```diff
@@ -122,7 +122,7 @@ def test_this_file(fname, test_file):
         with open(path, "r", encoding="utf-8") as file:
             test_this_file(path, file)
 
-    return [ msg % (relpath(fname, TOP_PATH), line_no) for (msg, fname, line_no) in errors ]
+    return [ msg.format(path=relpath(fname, TOP_PATH), line_no=line_no) for (msg, fname, line_no) in errors ]
```

Comment:
Maybe use `relative_to()` if we are're already using `pathlib`.

Reply:
Also just FYI: trying to use `relative_to` raises an error since `TOP_PATH` is absolute:
```
E           ValueError: 'tests/codebase/test_code_quality.py' is not in the subpath of '/Users/bryan/work/bokeh' OR one path is relative and the other is absolute.
```
In any case it just seems redundant/unecessary regardless